eclass/meson.eclass

```diff
@@ -283,6 +283,36 @@ meson_feature() {
 # variables. Invoke via "${mesonargs[@]}" in the calling environment.
 # This function is called from meson_src_configure.
 setup_meson_src_configure() {
+	local ltoflags=()
```

Comment:
Is there a reason to introduce a separate `ltoflags` for this and not simply put it straight to `mesonargs`? I don't think the option order really matters here.

Reply:
I am not sure why I did that but I think it may be a side effect of having refactored code around without thinking too deeply about whether the order matters.

Let me take another look.